src/liballoc/collections/vec_deque.rs

```diff
@@ -2043,6 +2049,146 @@ impl<T> VecDeque<T> {
         }
     }
 
+    /// Rearranges the internal storage of this deque so it is one contiguous slice, which is then returned.
+    ///
+    /// This method does not allocate and does not change the order of the inserted elements.
+    /// As it returns a mutable slice, this can be used to sort or binary search a deque.
+    ///
+    /// In case `self` is already contiguous, [`as_slices`](#method.as_slices) can be used to get immutable access.
```

Comment:
```suggestion
    /// Once the internal storage is contiguous, the [`as_slices`](#method.as_slices) and
    /// [`as_slices_mut`](#method.as_slices_mut) methods will return the entire contents of the
    /// `VecDeque` in a single slice.
```

Reply:
Did you accidentally revert this change?

Reply:
don't know, maybe :thinking:

Reply:
jup I did

Reply:
fixed